frontend/packages/ceph-storage-plugin/src/components/ocs-install/node-list.tsx

```diff
@@ -77,36 +85,55 @@ const getColumns = () => {
   ];
 };
 
+const hasTaintWithEffect = (taint: string, val: string) => _.isEqual(_.get(taint, 'effect'), val);
+
+const isNodeUnschedulable = (node: NodeKind) => {
+  const taints = _.get(node, 'spec.taints', []);
+  return taints.some(
+    (taint: string) =>
+      hasTaintWithEffect(taint, NO_SCHEDULE_EFFECT) ||
+      hasTaintWithEffect(taint, PREFER_NO_SCHEDULE_EFFECT) ||
+      hasTaintWithEffect(taint, NO_EXECUTE_EFFECT),
+  );
+};
+
+const hasOCSTaint = (node: NodeKind) => {
+  const taints = _.get(node, 'spec.taints', []);
```

Comment:
Move this to selectors

Reply:
> Move this to selectors

I'm trying to reduce our use of "selectors" because I don't think it's a good pattern, *especially* in conjunction with `_.get`, which is not type safe. There's really no need for a helper here. `node.spec` is always defined.